lib/src/Base/Graph/DrawableImplementation.cxx

```diff
@@ -1483,6 +1553,49 @@ Description DrawableImplementation::BuildDefaultPalette(const UnsignedInteger si
   return palette;
 }
 
+/* Build Tableau palette
+   Use 10 colors from Tableau palette.
+   Cycle through the hue wheel with 10 nuances and increasing darkness */
+Description DrawableImplementation::BuildTableauPalette(const UnsignedInteger size)
+{
```

Comment:
so this is the same as the tableau10 palette from matplotlib right ? that's very cool
can you attach a graph with these new colors ?

Reply:
Exactly. An example is done in python/doc/examples/graphs/use-color-palette.ipynb.